packages/redirect-parser/tests/helpers/main.ts

```diff
@@ -20,7 +20,7 @@ export const validateErrors = async function (t, { input, errorMessage }) {
   t.true(errors.some((error) => errorMessage.test(error.message)))
 }
 
-const parseRedirects = async function ({ redirectsFiles, netlifyConfigPath, configRedirects, minimal }) {
+export const parseRedirects = async function ({ redirectsFiles, netlifyConfigPath, configRedirects, minimal }: any) {
```

Comment:
Seems like we're now exporting both this function and `normalizeRedirect` and `validateErrors` and `validateSucsess` are now unused?
- https://github.com/netlify/build/blob/768626921b43de58d7191f01d3e941e8340790ce/packages/redirect-parser/tests/helpers/main.ts#L7-L21

Should we delete those?

Reply:
oh yea you are right forgot to delete them